life/updated/03-testimonies.adoc

```diff
@@ -3,39 +3,39 @@
 [.blurb]
 === A Testimony from several Friends belonging to the meeting at Hemel, Hemstead, and the parts adjacent, in the county of Hertford, concerning Elizabeth Stirredge, the 8th of the Third month, 1711.
 
-Our dear friends James and Elizabeth Stirredge, came here to reside about the year 1688,
+Our dear friends James and Elizabeth Stirredge came here to reside about the year 1688,
 whose coming into these parts was seasonable,
 and we believe ordered by the good providence of God.
-It had a tendency to the strength and edification of Friends;
+It had a tendency to the strength and edification of Friends,
 for about that time some were busy to weaken love among brethren,
 under specious pretenses of liberty, etc.
-But the said Elizabeth being gifted for the ministry,
+But the said Elizabeth, being gifted for the ministry,
 and acquainted with the wiles and subtle devices of the enemy,
-was made a serviceable instrument in our defense and preservation;
+was made a serviceable instrument in our defense and preservation,
 being attended with power, wisdom,
 and true zeal for the prosperity of the truth and people of God.
-She was sound and savory in her doctrine and public ministry,
+She was sound and savory in her doctrine and public ministry
```

Comment:
i'd keep the comma here after ministry,  (most of the commas you take out I agree with. But if here there is a change of theme. She was sound and savory in her doctrine and public ministry, and [she was] tender and affectionate in Christian advice and.... See what I mean? I'm not suggesting you write in the words "she was", just showing you that it's almost like a second sentence. 👍🏻